### ISSUE TITLE
Outdated dependencies

### ISSUE THREAD

**Issue by athyuttamre** [Quramy/typed-css-modules#7]
Installing the modules gives me a few dependency warnings, that could be fixed:

```
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated lodash@1.0.2: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
```

**Comment by Quramy** [comment]
They are transient dependencies. My module depends on only the following modules:

```
    "chalk": "^1.1.1",
    "css-modules-loader-core": "^1.0.0",
    "gaze": "^0.5.2",
    "glob": "^6.0.4",
    "is-there": "^4.3.1",
    "mkdirp": "^0.5.1",
    "yargs": "^3.32.0"
```

How do I fix it ?

**Comment by athyuttamre** [comment]
Sorry for the late reply here. All of these can be fixed by updating `gaze` to 1.1.1. Could you fix that and publish an update please? Thanks!

**Comment by Quramy** [comment]
@athyuttamre Thanks, I'll fix it :)
